packages/api/package.json

```diff
@@ -35,9 +35,9 @@
   "scripts": {
     "build": "tsc",
     "build:watch": "tsc --watch",
-    "test": "ts-mocha --recursive ./src/**/*.test.ts",
-    "e2e": "ts-mocha --recursive ./src/**/*.e2e.ts",
-    "test:watch": "ts-mocha --recursive ./src/**/*.test.ts -w --watch--extensions ts",
+    "test": "ts-mocha --recursive=true ./src/libs/**/*.test.ts ./src/tables/**/*.test.ts", 
+    "e2e": "ts-mocha --recursive=true  ./src/**/*.e2e.ts",
+    "test:watch": "ts-mocha --recursive=true ./src/libs/**/*.test.ts ./src/tables/**/*.test.ts -w --watch--extensions ts",
```

Comment:
mocha was being configured incorrectly, for some reason its not recursing all dirs, so i specified the folders where tests are